STLYEGUIDE_SPRESS_UPDATE/src/sass/_init.scss

```diff
@@ -96,7 +96,7 @@ $text-color: $black;
 $background-color: $white;
 /* Link (a:) */
 $link-color: $gray-dark;
-$link-hover: $secondary;
+$link-hover: $gray-dark;
```

Comment:
@aCyborg what was the logic behind changing the default link hover color?

Reply:
@MichellanneLi I stripped out anything 'project' related, and primary colors were project specific. You'll notice that I've changed everything to the grayscale palette.